crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.Author(**author.dict())
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_author(db: Session, author_id: int):
+    return db.query(models.Author).filter(models.Author.id == author_id).first()
+
+
+def get_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def update_author(db: Session, author_id: int, author: schemas.AuthorCreate):
+    db_author = db.query(models.Author).filter(models.Author.id == author_id).first()
+    if db_author:
+        for key, value in author.dict().items():
+            setattr(db_author, key, value)
+        db.commit()
+        db.refresh(db_author)
+    return db_author
+
+
+def delete_author(db: Session, author_id: int):
+    db_author = db.query(models.Author).filter(models.Author.id == author_id).first()
+    if db_author:
+        db.delete(db_author)
+        db.commit()
+    return db_author
+
+
+def create_book(db: Session, book: schemas.BookCreate):
+    db_book = models.Book(**book.dict())
+    db.add(db_book)
+    db.commit()
+    db.refresh(db_book)
+    return db_book
+
+
+def get_book(db: Session, book_id: int):
+    return db.query(models.Book).filter(models.Book.id == book_id).first()
+
+
+def get_books(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Book).offset(skip).limit(limit).all()
+
+
+def update_book(db: Session, book_id: int, book: schemas.BookCreate):
```

Comment:
Consider using `schemas.BookUpdate` instead of `schemas.BookCreate` for the `update_book` function if you have a separate schema for updates. This would allow partial updates and better reflect the intention of updating an existing record.